live-site/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "dependencies": {
     "gatsby": "^4.20.0",
-    "gatsby-plugin-mailchimp": "^5.2.2"
-
+    "gatsby-plugin-mailchimp": "^5.2.2",
+    "react-device-detect": "^2.2.2"
```

Comment:
Remove this library - (I'm not sure if it needs to be rendered on mobile - check w/ sarah) and also if we do need to do a isMobile check we can use react-use-match-media like in other parts of our code